docs/src/content/docs/ja/guides/components.mdx

```diff
@@ -59,16 +59,24 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <Tabs>
-	<TabItem label="恒星">シリウス、ベガ、ペテルギウス</TabItem>
-	<TabItem label="衛星">イオ、エウロパ、ガニメデ</TabItem>
+	<TabItem label="恒星" icon="star">
```

Comment:
I cannot add a comment to the line for the description above the example but I think the description is missing the new additional line added to the description:

> Use the optional `icon` attribute to include one of [Starlight’s built-in icons](#all-icons) next to the label.

Reply:
Thanks! I've added the missing sentence 🙏